schedule/yast/minimalx_zvm.yaml

```diff
@@ -1,46 +0,0 @@
----
-name: minimalx
```

Comment:
it would help to share the knowledge to add an additional commit why this one is not scheduled anymore (or at least I could not find it) and its schedule is removed.